django/db/models/constraints.py

```diff
@@ -9,6 +10,7 @@
 from django.db.models.query_utils import Q
 from django.db.models.sql.query import Query
 from django.db.utils import DEFAULT_DB_ALIAS
+from django.utils.deprecation import RemovedInDjango51Warning
```

Comment:
Things deprecated in Django 5.1 will be remove in Django 6.0.
```suggestion
from django.utils.deprecation import RemovedInDjango60Warning
```

Reply:
:+1: but I thought it would land in version 5.0 ? (and thus should be removed in 5.2 ?)